gradle/testing/randomization/policies/solr-tests.policy

```diff
@@ -100,6 +100,9 @@ grant {
   permission java.lang.RuntimePermission "loadLibrary.jaas";
   permission java.lang.RuntimePermission "loadLibrary.jaas_unix";
   permission java.lang.RuntimePermission "loadLibrary.jaas_nt";
+  // needed by ONNX integration (TODO: there is a cleaner way to handle this)
+  permission java.lang.RuntimePermission "loadLibrary.*"; // TODO: make more specific
+  permission java.io.FilePermission "/Users/cpoerschke/opennlp-data/onnx/sentence-transformers/vocab.txt", "read"; // TODO: remove when no longer used
```

Comment:
I'd love to see the "films" [vector model](https://github.com/apache/solr/tree/main/solr/example/films/vectors) as convertible to ONNX and then we could use it e.g. for tests. But not yet looked into details much.